### ISSUE TITLE
Collaboration with existing repositories

### ISSUE THREAD

**Issue by suchitakulkarni** [nikosarcevic/HEP-ASTRO-COSMO#17]
Just want to point out that we should be careful with the scope of this listing and also manage collaboration with other existing repositories. e.g. I would link ASCL for as many astro tools as possible and send any astro tools nominations there. Same goes for neutrino codes and send those to Peter Denton who's managing it. 

What do the others think?

**Comment by nikosarcevic** [comment]
on one hand I do agree 100 and all the existing repos/links should be cited/listed. 

On the other hand I am just thinking that then we will have a nicely sorted HEP list and the rest (astro and cosmo) will be just jumbled under "here are links within links here go have at it". Hope I am making sense?  This all goes with the scope in the issue #9 @bradkav pointed out. We can discuss more.

**Comment by teuben** [comment]
It doesn't make sense to me to start yet another code registry , where ASCL already serves this purpose. More useful would be to have the codes listed here and not yet in ASCL, ot have them added to ASCL.   Arguably (but we are working on it) you cannot say "give me all Nbody codes in ASCL", or by some selected criterion.  
Equally useful is to add more computation specific keywords to the  AUT https://astrothesaurus.org/ and use them in something lilke ASCL.   This is something I'm thinking about.
If we have this, both @suchitakulkarni and @nikosarcevic will be happy.

**Comment by nikosarcevic** [comment]
> It doesn't make sense to me to start yet another code registry , where ASCL already serves this purpose. More useful would be to have the codes listed here and not yet in ASCL, ot have them added to ASCL. Arguably (but we are working on it) you cannot say "give me all Nbody codes in ASCL", or by some selected criterion.
> Equally useful is to add more computation specific keywords to the AUT https://astrothesaurus.org/ and use them in something lilke ASCL. This is something I'm thinking about.
> If we have this, both @suchitakulkarni and @nikosarcevic will be happy.

yeah good points there. let's keep it as is for now and see how it evolves. I would like to hear others to give their 2 cents as well and also to sleep on it
